index.html

```diff
@@ -157,7 +177,7 @@ <h1 class="contact-header">Contact me</h1>
                <input type="text" id="name" name="name-label" maxlength="30" placeholder="Enter your name" required>
                <input type="email" id="email" name="email-label"placeholder="Enter your email" required>
                <textarea id="textarea" name="text-area" maxlength="500" rows="6" cols="50" placeholder="Write your message here" required ></textarea>
-                <button class="btn-submit" type="submit">Get It Touch</button>
+                <button class="btn-submit" type="submit">Get In Touch</button>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give correct style to all buttons as shown in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Uy7TAUKVJbGSY3Ol-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/213466553-4b1ba499-4a6b-4f26-b819-42f0913840e9.png)